### PR TITLE
fixed formatting issues in MouseEvent section

### LINKED ISSUE #1: 5-interact Event Handlers and Listeners has some formatting problems...
While reading through this lab, I noticed some of the words that should be in monospace font are instead in regular font with a bunch of ` characters around them. I would like to fix this problem for my branch/PR assignment.

### PR DESCRIPTION
**Pull request by n-bakken**
fixes #1

### CODE REVIEW

File: docs/labs/5-Interact.md [hunk 1]
```diff
@@ -223,20 +223,20 @@ This parameter of type ```MouseEvent```
 is commonly named ```e``` to stand for the event.
 To make our program more exciting,
 let's move the ball to where our mouse location is.
-Inside the `mousePressed```` function,
-add a call to ```ball```'s ```setLocation```.
+Inside the `mousePressed` function,
+add a call to ```ball```'s `setLocation`.
```
Comment (onzfonz) [inline docs/labs/5-Interact.md:227]:
```suggestion
add a call to `ball`'s `setLocation`.
```

File: docs/labs/5-Interact.md [hunk 2]
```diff
@@ -223,20 +223,20 @@ This parameter of type ```MouseEvent```
...
 To set the location,
-we need to provide an ```x``` and a ```y```.
+we need to provide an `x` and a `y`.
 We will not move the ```x``` in this case,
-but just the ```y``` coordinate.
+but just the `y` coordinate.
 To keep the horizontal movement,
-ask the ball for its x location by calling ```getX()```
+ask the ball for its x location by calling `getX()`
```
Comment (onzfonz) [inline docs/labs/5-Interact.md:233]:
```suggestion
ask the ball for its `x` location by calling `getX()`
```

File: docs/labs/5-Interact.md [hunk 3]
```diff
@@ -223,20 +223,20 @@ This parameter of type ```MouseEvent```
...
 on the ball and then just set it back reflexively as its x location again.
 To get the y, however,
```
Comment (onzfonz) [inline docs/labs/5-Interact.md:235]:
```suggestion
To get the `y`, however,
```

**Review by onzfonz — CHANGES_REQUESTED** [review]

**Review by onzfonz — CHANGES_REQUESTED** [review]
you could have used the commit suggestion.
Please also edit the description to say fixes the issue instead of the title

**Comment by n-bakken** [comment]
I believe I changed the title/description accordingly, please let me know if I did this incorrectly!

File: docs/labs/5-Interact.md [hunk 4]
```diff
@@ -223,20 +223,20 @@ This parameter of type ```MouseEvent```
 is commonly named ```e``` to stand for the event.
 To make our program more exciting,
 let's move the ball to where our mouse location is.
-Inside the `mousePressed```` function,
-add a call to ```ball```'s ```setLocation```.
+Inside the `mousePressed` function,
+add a call to `ball`s `setLocation`.
```
Comment (onzfonz) [inline docs/labs/5-Interact.md:227]:
```suggestion
add a call to `ball`'s `setLocation`.
```

**Review by onzfonz — CHANGES_REQUESTED** [review]
just one more, please make sure to use the commit suggestion from the pull request itself instead of commit the code from the editor

**Review by onzfonz — APPROVED** [review]

**Pull request merged**